config/app/com/microsoft/azure/iotsolutions/uiconfig/services/external/PackageValidation/PackageValidatorFactory.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+package com.microsoft.azure.iotsolutions.uiconfig.services.external.PackageValidation;
+
+import com.microsoft.azure.iotsolutions.uiconfig.services.models.ConfigType;
+import com.microsoft.azure.iotsolutions.uiconfig.services.models.PackageType;
+
+public class PackageValidatorFactory
+{
```

Comment:
formatting {

Reply:
Yes as discussed we will do this in different PR.